aws/lambda/log-classifier/ruleset.toml

```diff
@@ -28,6 +28,10 @@ pattern = '^>>> Lint for.*'
 name = 'GHA timeout'
 pattern = '^##\[error\]The action has timed out.'
 
+[[rule]]
+name = 'Faulty ROCM runner'
+pattern = 'Error: .*[dD]etect.*GPUs on the runner'
```

Comment:
I'm new to writing log classifier rule, but does the log message just say `Failed to detect GPUs on the runner` without the `Error:` prefix?

Reply:
The example is like that, yes, but I changed the log message a bit in [this PR](https://github.com/pytorch/pytorch/pull/89011/files). This regex captures both potential versions of the log